dev-lang/zig/zig-9999.ebuild

```diff
@@ -23,22 +23,17 @@ RESTRICT="!test? ( test )"
 
 BUILD_DIR="${S}/build"
 
-# According to zig's author, zig builds that do not support all targets are not
-# supported by the upstream project.
-ALL_LLVM_TARGETS=(
-	AArch64 AMDGPU ARM AVR BPF Hexagon Lanai Mips MSP430 NVPTX
-	PowerPC RISCV Sparc SystemZ WebAssembly X86 XCore
-)
-ALL_LLVM_TARGETS=( "${ALL_LLVM_TARGETS[@]/#/llvm_targets_}" )
-LLVM_TARGET_USEDEPS="${ALL_LLVM_TARGETS[@]}"
```

Comment:
We don't need to repeat this:

https://github.com/gentoo/gentoo/blob/987aa47ed905da959213f9fa8ef95da25c00e8c7/profiles/base/package.use.force#L34-L40